### ISSUE TITLE
data & data_std not defined

### ISSUE THREAD

**Issue by yeahjack** [notsebastiano/GP_algorithm#2]
Hi, I was using your script to implement the GP algorithm, but I found variable "data" not defined in function td_embedding and variable "data_std" not defined in function grassberg_procaccia, and I do not know what these mean, please help!

**Comment by ricardofrantz** [comment]
hey, any update here?

**Comment by yeahjack** [comment]
nope... haven't got any feedbacks

**Comment by notsebastiano** [comment]
Hi, sorry I did not maintain this, I'll be updating this by tomorrow

**Comment by ricardofrantz** [comment]
Hi Sebastiano,

I am also new to this field and have used the routine according to this reference https://hess.copernicus.org/articles/22/5069/2018/.
It would be great if you could prove a functional example and if you could outline some comments on the choice of parameters (and maybe a reference). 
As far as I understand, when finding the Hausdorff dimension, we have to loop over a range of, say, m=1..10 until a reasonable convergence of d is reached.

Thank you for sharing your work with us,
R

**Comment by notsebastiano** [comment]
Hi @ricardofrantz , Hi @yeahjack I have updated the code and solved the issue with data_std.

I have also added a Notebook with a functional example on the Lorenz attractor. I use that dynamical system to first obtain a timeseries of scalars (as you would have from the real world in absence of the knowledge of the underlying deterministic dynamical model). Then in the example the GP algorithm is used to calculate the correlation dimension of the reconstructed phase space for different embedding dimensions (refer to takens delay embedding)  and you will se that after emb_dim = 3 the dimension is constant. This as an indicator that the dynamical manifold is unfolded as well reconstructed in 3 dims

Feel free to let me know if further problems arise, or doubts about the algorithm!
Sebastiano
